ethereumj-core/src/main/java/org/ethereum/solidity/compiler/SolidityCompiler.java

```diff
@@ -38,38 +47,96 @@ public SolidityCompiler(SystemProperties config) {
         solc = new Solc(config);
     }
 
-    public static Result compile(File sourceDirectory, boolean combinedJson, Options... options) throws IOException {
+    public static Result compile(File sourceDirectory, boolean combinedJson, Option... options) throws IOException {
         return getInstance().compileSrc(sourceDirectory, false, combinedJson, options);
     }
 
-    public enum Options {
+    public static final class Options {
```

Comment:
Could you, please, add comment here, that it's here for backward compatibility

Reply:
Will do.